bubblewrap.c

```diff
@@ -556,205 +567,205 @@ setup_newroot (bool unshare_pid,
             die_with_error ("Can't mkdir parents for %s", op->dest);
         }
 
-      switch (op->type) {
-      case SETUP_RO_BIND_MOUNT:
-      case SETUP_DEV_BIND_MOUNT:
-      case SETUP_BIND_MOUNT:
-        if (source_mode == S_IFDIR)
-          {
-            if (mkdir (dest, 0755) != 0 && errno != EEXIST)
-              die_with_error ("Can't mkdir %s", op->dest);
-          }
-        else
-          {
-            if (ensure_file (dest, 0666) != 0)
-              die_with_error ("Can't create file at %s", op->dest);
-          }
+      switch (op->type)
+        {
+        case SETUP_RO_BIND_MOUNT:
+        case SETUP_DEV_BIND_MOUNT:
+        case SETUP_BIND_MOUNT:
+          if (source_mode == S_IFDIR)
+            {
+              if (mkdir (dest, 0755) != 0 && errno != EEXIST)
+                die_with_error ("Can't mkdir %s", op->dest);
+            }
+          else if (ensure_file (dest, 0666) != 0)
+            die_with_error ("Can't create file at %s", op->dest);
```

Comment:
Here it converted 
 if {} else { if (foo) {} } 
to
if {} else if (foo) {}
I don't dislike the later in the right place, but i'm not sure always doing it is right.
But, i've been unable to make it not always do that.